_infra/helm/securebanking-openbanking-uk-rs/values.yaml

```diff
@@ -1,40 +1,47 @@
+---
+configmap:
+  apiVersion: v1
+  # Addition Spring properties can be provided as a yaml block here, this will be mounted into the container and override / extend the config found in the application.yml inside the jar
+  springConfig: {}
+
 deployment:
   apiVersion: apps/v1
-  replicas: 1
-
-  rollingUpdate:
-    maxSurge: 50%
-    maxUnavailable: 25%
-
-  imagePullPolicy: Always
-
-  # Deploy a different image to the intended release (useful for development)
-  imageOverride:
-    enabled: true
-    repo: eu.gcr.io/sbat-gcr-develop/securebanking/securebanking-openbanking-uk-rs
-    tag: latest
-    tagDelimiter: ":"
-
-  server:
-    port: 8080
-
+  containerPort: 8080
+  image:
+    # repo:
```

Comment:
Documentation again here to state these must be provided, form they take, what they are used for etc.